reckon-gradle/src/main/java/org/ajoberstar/reckon/gradle/ReckonExtension.java

```diff
@@ -15,16 +15,36 @@
  */
 package org.ajoberstar.reckon.gradle;
 
+import java.util.Arrays;
+import java.util.Optional;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Objects'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.ajoberstar.com/coding_rules#rule_key=squid%3AUselessImportCheck)